tests/unit/amo/test_installAddon.js

```diff
@@ -1092,6 +1126,7 @@ describe(__filename, () => {
         });
       });
 
+      // FIXME
```

Comment:
Should we skip + add an invariant ? I'm not sure this is possible to reach `install()` with a `null` add-on from a user perspective.

Reply:
Yeah, I think the button will be disabled, so it shouldn't be possible. Replacing with an `invariant` makes sense.